js/GraphPane.js

```diff
@@ -120,21 +120,18 @@ export default function GraphPane({ graph, ...props }) {
     .reverse();
 
   return <Pane {...props}>
-    Include:
     <DepInclude type='dependencies' />
     <DepInclude type='devDependencies' />
     <DepInclude type='peerDependencies' />
 
     <label>
-      Colorize by:
-      <select value={colorize ?? ''} onChange={e => setColorize(e.target.value)}>
+      <select value={colorize ?? ''} onChange={e => setColorize(e.target.value)} style={{ borderRadius: '0.5em', padding: 4, marginTop: '0.5em' }}>
+        <option value=''>Colorize By ...</option>
```

Comment:
This change is unrelated and also it means we'll have a dropdown with the text "NPMS.io overall score" without a label describing what that is

Reply:
The initial value is "Colorize By ...", so users will see that when the page loads.  They'll also see it as the top option when the drop-down is opened.

Reply:
I know what you mean, but this remains a label-less field. https://www.nngroup.com/articles/form-design-placeholders/